airbyte-integrations/connectors/source-mailchimp/source_mailchimp/manifest.yaml

```diff
@@ -0,0 +1,253 @@
+version: 0.51.2
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - lists
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path:
+        - '{{ parameters.field_path }}'
+  basic_authenticator:
+      type: BasicHttpAuthenticator
+      password: '{{ config.credentials.apikey }}'
+      username: 'anystring'
+  oauth_authenticator:
+      type: BearerAuthenticator
+      api_token: '{{ config.credentials.access_token }}'
+  requester:
+    type: CustomRequester
+    class_name: source_mailchimp.components.MailchimpRequester
+    http_method: GET
+    request_headers: {}
+    path: '{{ parameters.path }}'
+    request_parameters:
+      sort_dir: ASC
+      sort_field: '{{ parameters.cursor_field }}'
+    authenticator:
+      class_name: source_mailchimp.components.MailchimpAuthenticator
+      basic_auth: "#/definitions/basic_authenticator"
+      oauth: "#/definitions/oauth_authenticator"
+    request_body_json: {}
```

Comment:
Sorry the missing return here @topefolorunso right now I'm talking to our engineer team about the declaration of nested custom components. Which is the case for this one.Any updates return to you here asap.